guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -2,15 +2,21 @@
 
 = Configuring Repositories
 
-Use this procedure to enable the repositories that are required to install {ProductName}. Choose from the available list which operating system and version you are installing on:
+Use this procedure to enable the repositories that are required to install {ProductName}.
+Choose from the available list which operating system and version you are installing on:
 
 ifdef::foreman-el,katello[]
 * xref:#repositories-centos-8[CentOS 8]
 * xref:#repositories-centos-7[CentOS 7]
 endif::[]
 ifdef::foreman-el,katello,satellite[]
+<<<<<<< HEAD
 * xref:#repositories-rhel-8[{RHEL} 8]
 * xref:#repositories-rhel-7[{RHEL} 7]
+=======
+* xref:#repositories-rhel-7[Red Hat Enterprise Linux 7]
+//* xref:#repositories-rhel-8[Red Hat Enterprise Linux 8]
```

Comment:
Is this meant to be a comment? We generally don't add comments like this. Either we include it, or not.